packages/xrpl/src/sugar/autofill.ts

```diff
@@ -230,13 +230,13 @@ export async function setNextValidSequenceNumber(
 }
 
 /**
- * Fetches the account deletion fee from the server state using the provided client.
+ * Fetches the owner reserve fee from the server state using the provided client.
  *
  * @param client - The client object used to make the request.
  * @returns A Promise that resolves to the account deletion fee as a BigNumber.
```

Comment:
Please update the references to `Account Deletion` into Owner reserve in these comments